six/modules/c++/six.sicd/source/ImageFormation.cpp

```diff
@@ -24,6 +24,24 @@
 using namespace six;
 using namespace six::sicd;
 
+
+bool Distortion::operator==(const Distortion& rhs) const
```

Comment:
Not directly applicable to this pull request but might as well fix it at the same time... `Distortion`'s constructor isn't very useful.  It should be default-constructing everything to the reserved uninitialized value.  Go ahead and add that in.